server/sonar-web/src/main/js/apps/quality-profiles/views/RestoreProfileView.js

```diff
@@ -37,28 +36,20 @@ export default ModalFormView.extend({
           this.ruleSuccesses = r.ruleSuccesses;
           this.ruleFailures = r.ruleFailures;
           this.render();
-          this.addProfile(r.profile);
+          this.trigger('done');
         })
         .catch(e => {
           this.enableForm();
           e.response.json().then(r => this.showErrors(r.errors, r.warnings));
         });
   },
 
-  addProfile (profileData) {
-    const profile = new Profile(profileData);
-    this.collection.add([profile], { merge: true });
-    const addedProfile = this.collection.get(profile.id);
-    if (addedProfile != null) {
-      addedProfile.trigger('select', addedProfile);
-    }
-  },
-
   serializeData() {
-    return Object.assign({}, ModalFormView.prototype.serializeData.apply(this, arguments), {
+    return {
+      ...ModalFormView.prototype.serializeData.apply(this, arguments),
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use the rest syntax to declare this function's arguments. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS3513)